src/drive/mobile/lib/cozy-helper.js

```diff
@@ -44,13 +44,19 @@ export const initBar = async client => {
   if (document.getElementById('coz-bar')) {
     return
   }
+
+  let token = null
+  try {
+    token = await getToken()
+  } catch (_) {}
```

Comment:
Are we sure we want to completly silent the catched exception?

To avoid the let declaration:

```js
// somewhere
getTokenWithNoException = async () => {
    try {
        return await getToken()
    } catch (_) {}
}

// and in your function

const token = await getTokenWithNoException()
cozy.bar.init({...options, token})
```

Reply:
The separate function makes sense here, I'll do that! But yes we want to silence the exception, not being abble to fetch a token for the bar has no real consequences — it will work offline as best as it can, and that's it.